### PR TITLE
Add language toggle changes

### PR DESCRIPTION
**Pull request by Joanneyoung01**
### What changed

- Add the updated language toggle to base files
- Add currentUrl local variable 
- Add peer dependancy

https://govukverify.atlassian.net/browse/DFC-579

### Why did it change

As part of the changes [required for the new language toggle, ](https://govukverify.atlassian.net/wiki/spaces/DIFC/pages/4148822329/Language+toggle+update)
 
### Issue tracking

https://govukverify.atlassian.net/browse/DFC-579

## Checklists

This has been tested on passport and driving license locally, with particular focus on on ensure that any existing parameters are persisted - evidence attached

https://github.com/govuk-one-login/ipv-cri-common-express/assets/38694448/34bb34c4-f584-412b-88c7-b6bc9633eee2

https://github.com/govuk-one-login/ipv-cri-common-express/assets/38694448/0847598e-9107-40d6-a4cb-dbbbb90062a6

### CODE REVIEW

**Review by di-aholme — DISMISSED** [review]
Verified that changes are working locally via pulling this branch and manually adding this code to the node module for Address CRI and DL CRI

**Review by merlinc — CHANGES_REQUESTED** [review]
The URL should be constructed using built in Node functionality.

`cri-common-express` package is built is using `npm` not `yarn` - so `yarn.lock` should be removed.

File: src/lib/locals.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ module.exports = {
     const toggleValue = req.app.get("APP.LANGUAGE_TOGGLE_ENABLED");
     res.locals.showLanguageToggle = toggleValue && toggleValue === "1";
     res.locals.htmlLang = req.i18n.language;
+    res.locals.currentUrl =
```
[the remark this answers is not in the document]
Comment (di-aholme) [reply src/lib/locals.js]: We've released a new version of the language toggle to allow the URL parsing to occur here when the variable is declared

**Review by merlinc — APPROVED** [review]
Approving, because this is an improvement and I don't want to hold this up.

But URL, can throw an error, so ideally that would be specifically caught and handled, rather than waiting for the middleware to throw an error and the global error handler catching it.

**Pull request merged**